app/helpers/users_helper.rb

```diff
@@ -7,7 +7,7 @@ def gravatar_for user, options = {size: Settings.avt.default_size}
   end
 
   def check_admin user
-    return unless !current_user.admin? && current_user?(user)
+    return unless current_user.admin? && !current_user?(user)
```

Comment:
tại sao hàm này check admin mà lại là unless current_user.admin? vậy em dùng if chứ nhỉ
```suggestion
    return if current_user.admin? && !current_user?(user)
```

Reply:
Dạ câu lệnh này là check xem có phải là admin không thì để hiển thị ra nút xóa ạ. Lúc đầu em để if nhưng mà chạy rubucop không pass được nên phải chuyển qua unless chị. Nếu không phải là user và không phải là người dùng hiện tại thì không làm gì cả.